docs/pages/deployment/docker.rst

```diff
@@ -0,0 +1,131 @@
+.. _running-docker:
+
+Running on Docker
+#################
+
+This guide helps you to configure the Nuts node in Docker.
+To use the most recent release use ``nutsfoundation/nuts-node:latest``. For production environments it's advised to use a specific version.
+
+Examples
+********
+The examples assume the current working directory on the host contains the following structure:
+
+.. code-block:: none
+
+    .
+    ├── config
+    │   └── nuts.yaml
+    └── data
+
+The ``data`` directory must be owned by container user ``18081``. (``chown -R 18081:18081 ./data``)
+
+These examples use the following ``nuts.yaml`` configuration file:
```

Comment:
Maybe we should simplify this by not using the YAML file, but environment variables?